pages.es/common/[.md

```diff
@@ -0,0 +1,25 @@
+# [
+
+> Evalúa una condicion.
+> Retorna 0 si la condición se cumple, 1 si esta no se cumple.
+> Más información: <https://www.gnu.org/software/coreutils/test>.
+
+- Prueba si una variable dada es igual a una cadena dada:
+
+`[ "{{MI_VARIABLE}}" == "{{/bin/zsh}}" ]`
+
+- Prueba si una variable dada está vacía:
+
+`[ -z "{{BRANCH_DE_GIT}}" ]`
+
+- Prueba si un archivo existe:
+
+`[ -f "{{ruta/al/archivo_o_directorio}}" ]`
```

Comment:
This is in the main page, but does this really support directories too? :thinking:

Reply:
Yep, you're right.  This should also be fixed in the English page.
```suggestion
`[ -f "{{ruta/al/archivo}}" ]`
```